client/src/pages/record.tsx

```diff
@@ -167,6 +214,65 @@ function RecordPage(props: {
   const warnEmptyTranscript =
     curAnswer?.attentionNeeded === AnswerAttentionNeeded.NEEDS_TRANSCRIPT;
 
+  const [transcriptText, setTranscriptText] = useState<string>("");
+  const [editorState, setEditorState] = useState<EditorState>(
+    EditorState.createWithContent(ContentState.createFromText(""))
+  );
+  const markdownConfig = {
+    blockTypesMapping: {
+      /* mappings */
+    },
+    emptyLineBeforeBlock: true,
+  };
+
+  // Used to get the initial plain text
+  function getTranscriptText(text: string) {
+    setTranscriptText(text);
+    const _editorState = EditorState.createWithContent(
+      ContentState.createFromText(text)
+    );
+    setEditorState(_editorState);
+    return transcriptText;
+  }
+
+  // Get markdown from editor state
+  function getMarkdown(contentState: ContentState) {
```

Comment:
try a more telling name, something like getMarkdownFromEditor

the usual gist is that if you have to leave a comment of what the function does, then the name of the function isn't good enough

Reply:
Thanks for letting me know this as well. I currently have the comments in here since it makes it easy to tell where I've made changes.